### PR TITLE
Clean up SVG

### PR DESCRIPTION
**Pull request by techknowlogick**
The previous SVG was a combination of objects and paths. This PR cleans up the SVG.

### CODE REVIEW

**Comment by silverwind** [comment]
That inner border did look kind of nice on big logos, but I guess flat is the trend. Is the color change intentional?

Also, the new SVG does not include the `detail-remove` class anymore and I generally think we can drop that mechanism and you should apply this patch to do so:

<details><summary>Patch</summary>

```patch
diff --git a/Makefile b/Makefile
index 3c1304575..d5b707372 100644
--- a/Makefile
+++ b/Makefile
@@ -686,9 +686,9 @@ generate-gitignore:


 .PHONY: generate-images
 generate-images:
-	npm install --no-save --no-package-lock xmldom fabric imagemin-zopfli
+	npm install --no-save --no-package-lock fabric imagemin-zopfli
 	node build/generate-images.js

 .PHONY: pr\#%
 pr\#%: clean-all
diff --git a/build/generate-images.js b/build/generate-images.js
index 9b7b82017..c7f58f61d 100755
--- a/build/generate-images.js
+++ b/build/generate-images.js
@@ -2,9 +2,8 @@
 'use strict';

 const imageminZopfli = require('imagemin-zopfli');
 const {fabric} = require('fabric');
-const {DOMParser, XMLSerializer} = require('xmldom');
 const {readFile, writeFile} = require('fs').promises;
 const {resolve} = require('path');
 const Svgo = require('svgo');

@@ -39,25 +38,9 @@ async function generateSvgFavicon(svg, outputFile) {
   const {data} = await svgo.optimize(svg);
   await writeFile(outputFile, data);
 }

-async function generate(svg, outputFile, {size, bg, removeDetail} = {}) {
-  const parser = new DOMParser();
-  const serializer = new XMLSerializer();
-  const document = parser.parseFromString(svg);
-
-  if (removeDetail) {
-    for (const el of Array.from(document.getElementsByTagName('g') || [])) {
-      for (const attribute of Array.from(el.attributes || [])) {
-        if (attribute.name === 'class' && attribute.value === 'detail-remove') {
-          el.parentNode.removeChild(el);
-        }
-      }
-    }
-  }
-
-  svg = serializer.serializeToString(document);
-
+async function generate(svg, outputFile, {size, bg}) {
   const {objects, options} = await loadSvg(svg);
   const canvas = new fabric.Canvas();
   canvas.setDimensions({width: size, height: size});
   const ctx = canvas.getContext('2d');
@@ -92,9 +75,9 @@ async function main() {
   await generate(svg, resolve(__dirname, '../public/img/gitea-512.png'), {size: 512});
   await generate(svg, resolve(__dirname, '../public/img/gitea-192.png'), {size: 192});
   await generate(svg, resolve(__dirname, '../public/img/gitea-sm.png'), {size: 120});
   await generate(svg, resolve(__dirname, '../public/img/avatar_default.png'), {size: 200});
-  await generate(svg, resolve(__dirname, '../public/img/favicon.png'), {size: 180, removeDetail: true});
+  await generate(svg, resolve(__dirname, '../public/img/favicon.png'), {size: 180});
   await generate(svg, resolve(__dirname, '../public/img/apple-touch-icon.png'), {size: 180, bg: true});
 }

 main().then(exit).catch(exit);
```
</details>

**Review by 6543 — COMMENTED** [review]
the new color is brighter - should we realy change it?

**Comment by silverwind** [comment]
I'm fine with the color change personally.

File: assets/logo.svg [hunk 1]
```diff
@@ -1,161 +1,36 @@
...
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="main_outline" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
+	 y="0px" viewBox="0 0 640 640" style="enable-background:new 0 0 640 640;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#FFFFFF;}
+	.st1{fill:#73A952;}
+</style>
```
Comment (silverwind) [inline assets/logo.svg]: Can you move those fills to the `path` elements as `fill` attribute and then remove `class` on them?

**Comment by lunny** [comment]
@techknowlogick Could you paste some screenshots?

**Comment by silverwind** [comment]
@lunny click "Display Rich Diff" on the images in the diff.

**Comment by silverwind** [comment]
Looking at the diff once more, I can't help but feel the new logo is a downgrade. I think the new color looks a bit dull compared to the older more vibrant one and the removal of the border makes it seem lower quality as well to me.

Also, the "zoom in" will cause issues in systems like Discord that force a circle clipping of the image, I'd say we even need to zoom out the old logo a tiny bit further to prevent such clipping.

image

**Comment by techknowlogick** [comment]
For those who were unsure about the colour change, I've changed the colour back to the original. As for things such as discord clipping we can generate an image that that has appropriate image dimensions as different sites require different dimension requirements.

Screen Shot 2020-12-01 at 9 54 33 PM

**Comment by lunny** [comment]
The new logo looks bigger than the old one and lack of border.

**Comment by silverwind** [comment]
Why not zoom out the SVG so it fits in a circle around the outer edges?

**Comment by techknowlogick** [comment]
After having to edit a unique avatar sizing for discord, twitter, reddit, and github, I can tell you two things 1. I reaally dislike forcing circle avatars on users, and 2. by having the SVG as we have it now, allows us to scale the logo as needed, and not worry about extra spacing when calculating things.

**Comment by silverwind** [comment]
Hmm yeah I guess we want maximum available area, for things like favicon.

**Review by 6543 — APPROVED** [review]

**Comment by kdumontnu** [comment]
@techknowlogick @silverwind is this PR still active/under review? It's blocking my PR. Anything I can do to help?

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
No, seems good now. I didn't see it was ready.

**Pull request merged**